paddle/fluid/operators/beam_search_op_test.cc

```diff
@@ -30,23 +30,23 @@ using std::endl;
 
 void CreateInput(LoDTensor* ids, LoDTensor* scores) {
   LoD lod;
-  vector<size_t> level0({0, 2, 4});
-  vector<size_t> level1({0, 1, 2, 3, 4});
+  vector<size_t> level0{0, 2, 4};
+  vector<size_t> level1{0, 1, 2, 3, 4};
   lod.push_back(level0);
   lod.push_back(level1);
   ids->set_lod(lod);
   scores->set_lod(lod);
 
-  auto dims = framework::make_ddim(vector<int64_t>({4, 3}));
+  auto dims = framework::make_ddim(vector<int64_t>{4, 3});
   ids->Resize(dims);
   scores->Resize(dims);
   CPUPlace place;
 
   auto* ids_data = ids->mutable_data<int64_t>(place);
   auto* scores_data = scores->mutable_data<float>(place);
-  vector<int64_t> _ids({4, 2, 5, 2, 1, 3, 3, 5, 2, 8, 2, 1});
```

Comment:
this syntax have issues? really?  it seems init vector from a initialize_list, quite meet the c++ standard in my mind.

Reply:
Would rollback.